rpcs3/rpcs3qt/persistent_settings.h

```diff
@@ -14,7 +14,8 @@ namespace gui
 		const QString last_played = "LastPlayed";
 
 		// Date format
-		const QString last_played_date_format = "MMMM d yyyy";
+		const QString last_played_date_format_text   = "MMMM d yyyy";
```

Comment:
```suggestion
		const QString last_played_date_format_text   = "MMMM d, yyyy";
```
Is this possible? The US date format demands a comma before the year.

Reply:
done